.github/workflows/release.yml

```diff
@@ -0,0 +1,39 @@
+name: Release
+
+on: 
+  push:
+    tags:
+      - '!player/'
+      - 'v*'
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
```

Comment:
If you start it manually (which is good IMHO) what version does it release?

Reply:
This was a copy & paste mistake. As you aren't able to create manual releases (without a tag) today, I removed this option as well in 419033e5